### DISCUSSION TITLE
2023-08-08

### DISCUSSION THREAD

**Discussion by MoonJeWoong** [woowacourse-teams/2023-haru-study#282] [daily_meeting]
### 오늘의 할 일
마코,테오 - OAuth 로그인 기능 구현
히이로, 모디 - Logback 로깅 기능 구현 + 그라파나, 프로메테우스 조사
프론트 - OAuth 구현 조사 및 모달 스텝퍼 발행된 버그 중 스프린트 내 해결할 것들 선정

점심식사 이후 로그인 회의 짧게 가져가기

### 언젠가 할 일
- 소켓, 폴링 양방향 통신 어떻게 할지 (4차 스프린트 내에 그냥 진행하고 발표만 안하는 방향으로 갈건지 로그인 이후 결정하기)
- 질문 PlaceHolder로 유의미한 응답 예시를 제공하는 기능 (목표설정 가이드라인)
- 브랜치 전략 결정하기(dev -> main)
- A라는 스터디를 참여후 B스터디 참여한 뒤 다시 A스터디에 참여할 시 닉네임을 찾지 못한 이슈(로그인이 된다면 해결될 문제이긴 함)

### 개인 일정
- 근로 5:00 ~ 6:00 (노아)

**Comment by MoonJeWoong** [comment]
### 체크인
9점 - 아침에 로그인 과정 정리하면서 재밌었어서 10점인데 배고파서 1점 깎았어용... 점심 뭐먹죠?

### 체크아웃 
9점 피곤하긴 하지만 기본적인 로깅도 완료하고 프로케테우스도 띄워서 기분이 좋습니다!

**Comment by nlom0218** [comment]
### 체크인

10점 - 핸드폰이 망가져서 새로 사야돼요 그래서 기뻐요

### 체크아웃

9점 - 갑자기 핸도폰이 잘 되네요.. -1

**Reply by aak2075** [reply]
와 그럼 새로 샀을 때 또 망가뜨리면 기분 좋아지는건가요?

**Reply by woosung1223** [reply]
365일 기분 좋게 해드리겠습니다.

**Reply by nlom0218** [reply]
저혼자 행복할 순 없어요!

**Comment by aak2075** [comment]
### 체크인

10점 - OAuth 구현할 생각에 신나네요

### 체크아웃

10점 - OAuth 재밌네요

**Comment by woo-jk** [comment]
### 체크인
7점 - 집이 좋았는데..

### 체크아웃
6점 - 타이머가 마음대로 안돼요

**Comment by jaehee329** [comment]
### 체크인
7점 - 집이 좋았는데...

### 체크아웃
8점 - 극적으로 프로메테우스 콘솔이 떠요 ㅠㅠ +1점 했습니다

**Comment by woosung1223** [comment]
### 체크인
8점 - 오전에 강의가 있어서 -2점 ..

### 체크아웃
9점 - 은근 oauth 많이 해서 좋습니다 ㅎㅎ

**Comment by yeopto** [comment]
### 체크인

5점 - 피곤합니다

### 체크아웃

5점 - 할게 많습니다~~~~
